### ISSUE TITLE
API Enhancements/Missing Features

### ISSUE THREAD

**Issue by Fidgetcetera** [glitch-soc/mastodon#30]
Currently, at the very least, the following things are not served by the API and could be to make things a little nicer on clients:
The default post setting for your account
The character limit for posts on an instance
Muted conversations
Regex filtering/keywords 
The maximum size of files for media posts
Information about the instance/the API? (this was something I believe @marrus-sh kinda brought up in discord as maybe being given somehow elsewise but I didn't totally get, please feel free to input on that one and I can edit or disregard this portion as necessary!) 

Any additions are welcome, this was a result of my searching through discord for everything people had complained about not being available.

**Comment by marrus-sh** [comment]
Information about the instance can be provided by the API through `/api/v1/instances`, although currently this file isn't used for much aside from the instance title and description iirc.

**Comment by Fidgetcetera** [comment]
Right, I think the talk was about having stuff from /about/more actually be given?
